### PR TITLE
✨amp-recaptcha-input: Added Origin Trial Support for the experiment 

### PR DESCRIPTION
**Pull request by torch2424**
This adds support for origin trials for amp-recaptcha-input, as well as the ability to continue support for manually enabling the experiment with `AMP.toggleExperiment`. Also, adds a test for the experiment, and throws a user error if not enabled for easier debugging 😄 

[Referenced Code Example from amp-list](https://github.com/cathyxz/amphtml/blob/0a2293bf051380b0c38908727318ca1f356e74f2/extensions/amp-list/0.1/amp-list.js#L182)

# Examples

(After modifying the `test/unit/test-origin-experiments` for my desired domain, token, and experiment)

screen shot 2019-02-14 at 2 51 41 pm

(Before AMP.toggleExperiment)

screen shot 2019-02-14 at 3 30 12 pm

(After AMP.toggleExperiment)

screen shot 2019-02-14 at 3 30 35 pm

### CODE REVIEW

File: extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js [hunk 1]
```diff
@@ -146,6 +151,27 @@ export class AmpRecaptchaInput extends AMP.BaseElement {
...
+    if(isExperimentOn(this.win, 'amp-recaptcha-input')) {
+      return Promise.resolve(true);
+    }
+
+    // Check if we are enabled by an origin trial
+    installOriginExperimentsForDoc(this.getAmpDoc());
+    return originExperimentsForDoc(this.element)
+      .getExperiments()
+      .then(trials => {
+        return trials && trials.includes('amp-list-load-more');
```
Comment (cathyxz) [inline extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js]: Your experiment name is probably not `amp-list-load-more`. =)

**Comment by torch2424** [comment]
@cathyxz Ha! I totally changed in every place except where it mattered 😂 

Thank you!

File: extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js [hunk 2]
```diff
@@ -146,6 +151,27 @@ export class AmpRecaptchaInput extends AMP.BaseElement {
         ' and data-action attribute'
     ));
   }
+
+  /**
+   * Function to check if recaptcha experiment is enabled,
+   * through origin trial, or AMP.toggleExperiment
+   * @return {!Promise<boolean>}
+   */
+  _isExperimentEnabled() {
```
Comment (lannka) [inline extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js]: isExperimentEnabled_

Comment (torch2424) [reply extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js]: Oops, thanks for catching that. I do the _ in front of personal fields in personal projects. 😂

**Pull request merged**